docs.md

```diff
@@ -20,10 +20,9 @@ A Woodpecker workflow file could look like this:
 ```yaml
 steps:
   release-helper:
-    image: woodpeckerci/plugin-ready-release-go:0.6.1
```

Comment:
I'd prefer keeping a version in here - I think renovate should be able to take care of it like it does for other deps. This way, we encourage users to use a fixed tag instead of latest. What do you think?

Reply:
I changed it to `:<version>` for now. Feel free to test if renovate can catch it so we can use the latest version directly.